test/errorshow.jl

```diff
@@ -291,7 +291,7 @@ let undefvar
     @test occursin("DomainError with [0.0 -1.0 …", err_str)
 
     err_str = @except_str (1, 2, 3)[4] BoundsError
-    @test err_str == "BoundsError: attempt to access (1, 2, 3) at index [4]"
+    @test err_str == "BoundsError: attempt to access Tuple{$Int,$Int,$Int} at index [4]"
```

Comment:
Please rebase this (which will hopefully make the test failure go away), and add spaces after these commas (and in the other file as well). Hopefully then we can finally merge this. Sorry this has taken so long!

Reply:
@JeffBezanson Are you sure about the spaces after commas? If we are reverting the output of `summary` for tuples to use `typeof`, then the output has no commas, right? E.g., with Julia v1.5.1:
```julia
julia> typeof((1,2,3))
 Tuple{Int64,Int64,Int64} 
```
Or am I missing something?

Reply:
https://github.com/JuliaLang/julia/pull/37085

Reply:
OK I was not aware of that 😅 will change it!

Reply:
OK Done! 🙂

Reply:
@JeffBezanson is it OK this time?